packages/ckeditor5-dev-env/lib/release-tools/utils/transform-commit/getchangedfilesforcommit.js

```diff
@@ -14,12 +14,20 @@ const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
  * @returns {Array.<String>}
  */
 module.exports = function getChangedFilesForCommit( commitId ) {
-	const gitCommand = `git diff --name-only ${ commitId }~..${ commitId }`;
+	const gitCommand = `git log -m -1 --name-only --pretty="format:" ${ commitId }`;
 	const changedFiles = tools.shExec( gitCommand, { verbosity: 'error' } ).trim();
 
-	if ( !changedFiles ) {
-		return [];
```

Comment:
You can make this simpler:

![image](https://cloud.githubusercontent.com/assets/156149/26398878/4491f2e2-407b-11e7-88dd-316581c72917.png)

You don't need to check whether `\n\n` is in the string.